src/main/java/com/ivanmostovyi/demo/domain/Messages.java

```diff
@@ -0,0 +1,32 @@
+package com.ivanmostovyi.demo.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+@Table("message")
```

Comment:
`@Table("message")` is not required, because the table name will match class name